tests/data/config/mock_lfa.yaml

```diff
@@ -0,0 +1,15 @@
+# Default configuration for tests.
+s3instance: mock
```

Comment:
Picky comment consider adding quotes to `mock` -> `"mock"`.

Reply:
Done.